### PR TITLE
Feature #11

### LINKED ISSUE #11: Implement a mockup data generator
Developers need data in the data model to be able to test and run the application.
# User story

```
As a developer
I need mockup data for our data model
so that i can do use case and application tests
```
# Open Task
- [x] Compile a list of 'fake data' tool, preferably with Meteor packaging
# Task
- [x] Determine the Use Case Scenarios / Development processes for 'fake data' 
  - [x] Do research and get feedback
- [x] Choose a 'fake data' tool, preferably with Meteor packaging
# Definition of done
- [x] Packages installed and basic implementation created
# Resources

Faker.js comes as an NPM module. The following packages uses Faker.js in some way.
# Packages using faker.js (to our best knowledge)
- [digilord](https://github.com/digilord/meteor-faker) | server only
- [practical meteor](https://github.com/practicalmeteor/meteor-faker) | wrapper for Fake.js , client and server , seems good
- [schema faker](https://github.com/json-schema-faker/json-schema-faker) | seems great , uses schema definitions 
- [Anticoders](https://github.com/anticoders/meteor-fake)  | good for longer texts, lacks in other features
- [gbit:faker](https://github.com/girassolbit/meteor-faker) | wrapper Fake.js , client and server, has a Blaze helper, seems better

### PR DESCRIPTION
**Pull request by leosa83**
This closes #11

### CODE REVIEW

File: .meteor/packages [hunk 1]
```diff
@@ -27,4 +27,9 @@ twbs:bootstrap
 kadira:flow-router
 kadira:blaze-layout
 fortawesome:fontawesome
+<<<<<<< HEAD
```
Comment (brylie) [inline .meteor/packages:30]: Remove the conflict formatting to complete the merge:

```
<<<<<<< HEAD
```

File: .meteor/packages [hunk 2]
```diff
@@ -27,4 +27,9 @@ twbs:bootstrap
...
+practicalmeteor:faker
+dburles:factory
+=======
```
Comment (brylie) [inline .meteor/packages:33]: Remove the conflict formatting to complete the merge:

```
=======
```

File: .meteor/packages [hunk 3]
```diff
@@ -27,4 +27,9 @@ twbs:bootstrap
...
 simple:json-routes
+>>>>>>> develop
```
Comment (brylie) [inline .meteor/packages:35]: Remove the conflict formatting to complete the merge:

```
>>>>>>> develop
```

File: client/fakedata.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+            'closing'
+         ]);},
+         targetPlatforms: ['web'],
+         postalAddress: function(){ return {
+             city: faker.address.city(),
+             country:faker.address.country()
+         }},
+         links: function() { return [{url:faker.internet.url(),name:faker.lorem.sentence(),type:'web'},{url:faker.internet.url(),name:faker.lorem.sentence(),type:'facebook'},{url:faker.internet.url(),name:faker.lorem.sentence(),type:'twitter'}];},
+         dateListed: function() {return faker.date.past();}
+
```
Comment (brylie) [inline client/fakedata.js:27]: Cleanup empty lines

File: client/fakedata.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
+if(Meteor.isClient) {
```
Comment (brylie) [inline client/fakedata.js:1]: Move this file to the `projects` package folder.

**Comment by leosa83** [comment]
@brylie Should i revert and then make the changes or could i just make the changes to the files in develop and push directly?

**Comment by brylie** [comment]
Just open a new branch and apply fixes there. Lets make a practice of having PRs reviewed before merging.

**Comment by leosa83** [comment]
Ok. Yes that probably a good idea :)

**Comment by leosa83** [comment]
created #66 to handle your other comments

**Pull request merged**